packages/vue/src/router.ts

```diff
@@ -139,17 +143,17 @@ export function instrumentVueRouter(
     }
 
     if (options.instrumentPageLoad && isPageLoadNavigation) {
-      // eslint-disable-next-line deprecation/deprecation
-      const pageloadTransaction = getActiveTransaction();
-      if (pageloadTransaction) {
-        const existingAttributes = spanToJSON(pageloadTransaction).data || {};
+      const activeSpan = getActiveSpan();
+      const rootSpan = activeSpan && getRootSpan(activeSpan);
+      if (rootSpan) {
```

Comment:
m: Should we also check here if `op` of the span is `pageload` or `navigation`? This would be a bit more defensive, to avoid updating a user span maybe or something like this...?

Reply:
Yes this is a good idea. I'll make the change